survey_result_mail/models/survey_survey.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SurveySurvey(models.Model):
+    _inherit = "survey.survey"
+
+    send_survey_answers = fields.Boolean(
```

Comment:
Avoid plurals in names:

```suggestion
    send_result_mail = fields.Boolean(
```

Reply:
done